packages/swing-store/src/sqlStreamStore.js

```diff
@@ -148,23 +142,11 @@ export function sqlStreamStore(dbDir, io) {
     streamStatus.delete(streamName);
   };
 
-  const commit = () => {
-    if (db.inTransaction) {
-      db.prepare('COMMIT').run();
-    }
-  };
-
-  const close = () => {
-    // close without commit is abort
-    db.close();
-  };
-
   return harden({
     writeStreamItem,
     readStream,
     closeStream,
-    commit,
-    close,
+    dumpStreams,
```

Comment:
I think I would rather find a way for it not to be in the external API.  Since its only use is within the `swing-store` package (and for testing purposes at that), I wish there were some graceful way to avoid exposing it.

Reply:
Upon further reflection, it strikes me that, regardless of API factoring, the useful test in `test-state.js` would be to add some stream state to the test that exercises `getAllState` and `setAllState` and verify that they do the right thing.

Reply:
@FUDCo did we decide to remove `dumpStreams` from the `streamStore` returned as part of `makeSwingStore`, and move it to a `debug` component?

Reply:
see https://github.com/Agoric/agoric-sdk/pull/6722 for the `dumpsStreams` change